README.md

```diff
@@ -1,5 +1,13 @@
 # VGA Simulator
 
+**Note:** This project is not actively maintained. Consider using one of the following newer ports:
+
+- [Pedro, @pvieito](https://github.com/pvieito) made a CLI version of this app in Python that you can find in [this gist](https://gist.github.com/pvieito/8cdb54a9a03fd36e51c8df6e331a3006).
+- https://github.com/ARC-Lab-UF/vga-simulator is an actively maintained port of Pedro's Python version, with a beautiful and flexible command line interface.
```

Comment:
It probably still makes sense to have these below in the section they were before

Reply:
Just moved them back

Reply:
Under the "Other projects" heading makes sense. Looking good 👍